.github/README.md

```diff
@@ -0,0 +1,74 @@
+## Introduction
+
+- This repository is a fork of https://github.com/laravel/echo. 
```

Comment:
I think this list should be a paragraph, it seems a bit off to have the introduction as a set of bullet points.

Reply:
Fixed!